lib/board.rb

```diff
@@ -0,0 +1,50 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    self.cells = Array.new(9, " ")
+  end
+
+  def reset!
+    self.cells = Array.new(9, " ")
+  end
+  
+  # This #display method I essentially copy and pasted from the earlier tic-tac-toe projects before I learned about Enumberables and iterative methods like #each
+  def display
```

Comment:
it is generally good practice to leave inputs and outputs in a common interface file (like the new CLI class) where user interaction is expected.